src/database.ts

```diff
@@ -156,7 +167,7 @@ class Database extends ServiceObject {
       createMethod: (_, options, callback) => {
         return instance.createDatabase(formattedName_, options, callback);
       },
-    });
+    } as {} as ServiceObjectConfig);
```

Comment:
Curious why this is needed

Reply:
Ah, we don't have `requestModule` here. OK for now - we'll have to add types to `common-grpc`